tests/src/Common/test_dependencies/test_dependencies.csproj

```diff
@@ -6,8 +6,37 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <CLRTestKind>BuildOnly</CLRTestKind>
     <NugetTargetMoniker>.NETCoreApp,Version=v2.0</NugetTargetMoniker>
+    <NugetTargetMonikerShort>netcoreapp2.0</NugetTargetMonikerShort>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.CoreDisTools">
+      <Version>1.0.1-prerelease-*</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.Platforms">
+      <Version>4.4.0-$(CoreFxExpectedPrerelease)</Version>
```

Comment:
This version doesn't exist Microsoft.NETCore.Platforms is version 2.0.0 which is at least on of the current build failures. You can actually remove this reference completely as it will be pulled in transitively from Microsoft.Private.CoreFx.NETCoreApp.